integration/fetch_run_image_from_env.go

```diff
@@ -52,57 +39,29 @@ func testFetchRunImageFromEnv(t *testing.T, context spec.G, it spec.S) {
 			source, err = occam.Source(filepath.Join("testdata", "simple_app"))
 			Expect(err).NotTo(HaveOccurred())
 
-			nodeRunImage := settings.Metadata.Dependencies[0].Source
-
-			runNodeVersion, err := semver.NewVersion(settings.Metadata.Dependencies[0].Version)
-			Expect(err).NotTo(HaveOccurred())
-			runNodeMajorVersion := runNodeVersion.Major()
-
-			buildNodeVersion, err := semver.NewVersion(settings.Metadata.Dependencies[len(settings.Metadata.Dependencies)-1].Version)
-			Expect(err).NotTo(HaveOccurred())
-			buildNodeMajorVersion := buildNodeVersion.Major()
+			nodeRunImage := "this-is-a-run-image"
 
 			var logs fmt.Stringer
-			image, logs, err = pack.WithNoColor().Build.
+			_, logs, err = pack.WithNoColor().Build.
 				WithExtensions(
 					settings.Buildpacks.NodeExtension.Online,
 				).
 				WithBuildpacks(
 					settings.Buildpacks.NodeEngine.Online,
 					settings.Buildpacks.Processes.Online,
 				).
-				WithEnv(map[string]string{"BP_UBI_RUN_IMAGE_OVERRIDE": nodeRunImage, "BP_NODE_VERSION": fmt.Sprint(buildNodeMajorVersion)}).
+				WithEnv(map[string]string{"BP_UBI_RUN_IMAGE_OVERRIDE": nodeRunImage}).
 				WithPullPolicy("always").
 				Execute(name, source)
 
-			Expect(err).NotTo(HaveOccurred(), logs.String())
+			Expect(err).To(HaveOccurred())
```

Comment:
The function errors after the extension has run due to it tries to download the fake image `nodeRunImage := "this-is-a-run-image"`. Therefore the build fails, so I changed it to only check the logs and that the build failed instead of checking the logs and checking and that the build that succeeded. If we can find an image that will always be available, we can have also the build to succeed. In any case the extension is tested properly so to me seems like there is no issue of test coverage on this scenario.